kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -416,8 +416,10 @@ def _report_container_spec_metrics(self, pod_list, instance_tags):
                 if self.pod_list_utils.is_excluded(cid, pod_uid):
                     continue
 
-                tags = instance_tags[:]
-                tags += tagger.tag('%s' % replace_container_rt_prefix(cid), tagger.HIGH) or []
+                tags = tagger.tag('%s' % replace_container_rt_prefix(cid), tagger.HIGH) or None
```

Comment:
```suggestion
                tags = tagger.tag(replace_container_rt_prefix(cid), tagger.HIGH)
```

- `or None` seems not needed
- `'%s' %` seems not needed if `cid` is a string